corona-warn-app/cwa-app-android
# Personal Certificate data in error Report available (EXPOSUREAPP-9923)

## Testing
- App Info > Error Reports > Start
- scan certificate from multiple persons
- select one person as the main user
- delete all certificates from the selected user
- restart app
- open random certificate
- check if the error report contains sensitive data eg. 
`2021-10-07T14:33:31.472Z D/PersonCertificatesProvider: vaccPersons=[], tests=[], recos=[], cwaUser=CertificatePersonIdentifier(dateOfBirthFormatted=1982-02-11, lastNameStandardized=LIME, firstNameStandardized=MOTHER<MANDY)`

## Jira Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9923
